packages/react-devtools-extensions/package.json

```diff
@@ -37,6 +41,7 @@
     "web-ext": "^3.0.0",
     "webpack": "^4.41.2",
     "webpack-cli": "^3.3.10",
-    "webpack-dev-server": "^3.3.1"
+    "webpack-dev-server": "^3.3.1",
+    "windows-edge": "^1.0.1"
```

Comment:
New dependencies in `package.json` should be accompanied by an updated `yarn.lock` but I can always add this after the fact :)

Reply:
I have package-lock.json but there is no yarn.lock. I will see if I can generate yarn.lock and push with my next PR.

Reply:
No worries. I've already updated the Yarn lock in master.

To update it you'd need to use `yarn` to add new dependencies rather than NPM :)